<launch>
  <node pkg="state_machine" type="walk.py" name="smach_walk" />
  <node pkg="state_machine" type="time" name="time" />
  <include file="$(find robot)/launch/robot.launch" />
  <node pkg="rqt_smach" type="rqt_smach" name="rqt_smach" />
</launch>
